deploy/sample-backup-configs/sample_config_build_recipes.yaml

```diff
@@ -11,92 +11,92 @@
 # <NAME>:
 #    DESIGN: <>
 #    TARGET_CONFIG: <>
-#    deploy-triplet: null
+#    deploy_triplet: null
```

Comment:
since there's a fix to be made elsewhere anyway, can you also move `deploy_triplet` under `PLATFORM_CONFIG` like it used to be?

Reply:
Done